python/desc/imsim/optical_system.py

```diff
@@ -0,0 +1,317 @@
+"""
+Code to determine deviations in the Zernike coefficients determined by the
+LSST AOS closed loop control system. Results do not include contributions from
+the open loop lookup table.
+"""
+
+import os
+
+from astropy.io import fits
+import numpy as np
+from scipy.interpolate import interp2d
+from timeit import timeit
+
+from galsim.zernike import Zernike, zernikeBasis
+
+FILE_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
In keeping with conventions we've adopted from DM, I'd replace this with
```
import lsst.utils

FILE_DIR = lsst.utils.getPackageDir('imsim')
```
and I would suggest moving the `sim_data` directory into the `data` directory and rename it so that the subsequent paths are set as
```
AOS_PATH = os.path.join(FILE_DIR, 'data', 'optics_data', 'aos_sim_results.txt')
MATRIX_PATH = os.path.join(FILE_DIR, 'data', 'optics_data', 'sensitivity_matrix.txt')
```
etc..

Reply:
Just to make sure I understand, `os.path.join(FILE_DIR, 'data')` points to the `data` directory located at the root of this repository correct?

Reply:
Yes, that's right.